examples/bioasq9b.py

```diff
@@ -0,0 +1,232 @@
+participating at BioASQ please consult: 
+http://participants-area.bioasq.org/general_information/Task9b/
+
+Differences with BioASQ-training8b.json 
+	- 499 new questions added from BioASQ8
+		- The question with id 5e30e689fbd6abf43b00003a had identical body with 
+          5880e417713cbdfd3d000001. All relevant elements from both questions are available
+          in the merged question with id 5880e417713cbdfd3d000001.
+"""
+
+_HOMEPAGE = "http://participants-area.bioasq.org/datasets/"
+
+# Data access reqires registering with BioASQ.
+# See http://participants-area.bioasq.org/accounts/register/
+_LICENSE = ""
+
+_URLs = {"train": "BioASQ-training9b.zip", "test": "Task9BGoldenEnriched.zip"}
+
+
+class Bioasq9bDataset(datasets.GeneratorBasedBuilder):
+    """BioASQ Task B On Biomedical Semantic QA."""
+
+    VERSION = datasets.Version("1.0.0")
+
+    BUILDER_CONFIGS = [
+        datasets.BuilderConfig(
+            name="original",
+            version=VERSION,
+            description="Original BioASQ9 source schema.",
+        ),
+        datasets.BuilderConfig(
+            name="bigbio",
+            version=VERSION,
+            description="Simplified schema for BigScience-Biomedical QA tasks",
+        ),
+    ]
+
+    DEFAULT_CONFIG_NAME = "original"
+
+    def _info(self):
+
+        # BioASQ9 Task B original schema
+        if self.config.name == "original":
+            features = datasets.Features(
+                {
+                    "id": datasets.Value("string"),
+                    "type": datasets.Value("string"),
+                    "body": datasets.Value("string"),
+                    "documents": datasets.Sequence(datasets.Value("string")),
+                    "concepts": datasets.Sequence(datasets.Value("string")),
+                    "ideal_answer": datasets.Sequence(datasets.Value("string")),
+                    "exact_answer": datasets.Sequence(datasets.Value("string")),
+                    "triples": datasets.Sequence(
+                        {
+                            "p": datasets.Value("string"),
+                            "s": datasets.Value("string"),
+                            "o": datasets.Value("string"),
+                        }
+                    ),
+                    "snippets": datasets.Sequence(
+                        {
+                            "offsetInBeginSection": datasets.Value("int32"),
+                            "offsetInEndSection": datasets.Value("int32"),
+                            "text": datasets.Value("string"),
+                            "beginSection": datasets.Value("string"),
+                            "endSection": datasets.Value("string"),
+                            "document": datasets.Value("string"),
+                        }
+                    ),
+                }
+            )
+        # simplified scheam for QA tasks
+        elif self.config.name == "bigbio":
```

Comment:
e.g. `bigbio-qa` here instead of `bigbio` ... this would also allow some nice discovery by investigating the configs of the datasets